metaseq/model_parallel/criterions/vocab_parallel_cross_entropy.py

```diff
@@ -45,6 +44,7 @@ def forward(self, model, sample, reduce=True):
         loss = vocab_parallel_cross_entropy(net_output[0].float(), target)
         if has_pad:
             loss = loss * (target != self.padding_idx)
+        batch_loss = loss.mean(-1)
```

Comment:
I don't think we want this. We're going to be dividing by n twice now I believe?